nens_auth_client/views.py

```diff
@@ -150,7 +154,9 @@ def authorize(request):
       acceptable invitation.
     """
     client = get_oauth_client()
-    # client.check_error_in_query_params(request)
+    validation_errors = client.validate_authorize_request_params(request.GET)
```

Comment:
Note: underlying oauthlib also support request.POST, however I think we only are going to use that.